napari/plugins/io.py

```diff
@@ -35,12 +36,14 @@ def read_data_with_plugins(
 
     Parameters
     ----------
-    path : str
-        The path (file, directory, url) to open
+    paths : str, or list of string
+        The of path (file, directory, url) to open
     plugin : str, optional
         Name of a plugin to use.  If provided, will force ``path`` to be read
         with the specified ``plugin``.  If the requested plugin cannot read
         ``path``, a PluginCallError will be raised.
+    stack : bool
+        See `Viewer.open`
```

Comment:
i'm not in love with the `viewer.open` docstring either.  Might be simple enough just to save "if `True`, plugins should stack arrays found in `paths`"?

Reply:
You mean change it in the viewer's open method docstring or in all those intermediary functions ?